internal/pkg/cache/redis/redis.go

```diff
@@ -43,6 +44,22 @@ func NewRedis(address string, opts ...redis.DialOption) *Redis {
 	}
 }
 
+// NewRedis creates a new Redis instance.
```

Comment:
Can we do something like
```go
NewRedis (...) {
  cfg := &config.RedisConfig{ ... }
  return NewRedisWithConfig(cfg, opts...)
}
```

so that we can use the same construction code.

Reply:
I will look into this shortly.